tests/nominal/test_MeanResponseTransformer.py

```diff
@@ -1112,17 +1135,30 @@ def test_expected_output_binary_response(self, df, expected):
     )
     def test_expected_output_one_multi_level(self, df, expected):
         """Test that the output is expected from transform with a multi-level response and one level selected."""
-        x = MeanResponseTransformer(columns=["b", "f"], level=["blue"])
+        columns = ["b", "f"]
+        level = ["blue"]
+        x = MeanResponseTransformer(columns=columns, level=level)
+
+        for col in [
```

Comment:
could we move generation of this columns list to top of test and assign to an expanatory var name?  might make the test easier to read.  Same for other tests where you use this prefix + suffix format to generate column lists to iterate over

Reply:
good suggestion, tidied these up here https://github.com/lvgig/tubular/pull/174/commits